web/src/views/DandisetLandingView/Meditor.vue

```diff
@@ -228,6 +229,10 @@ export default defineComponent({
       return undefined;
     }
 
+    const options = computed(() => ({
```

Comment:
Why not keep the name `CommonVJSFOptions`? It'll make the intent and syntax below more clear

Reply:
See a851331e97bc80dd715fa4b581f897236e2ee994.